x-pack/plugins/fleet/server/services/package_policies_to_agent_permissions.ts

```diff
@@ -73,6 +73,13 @@ export async function storedPackagePoliciesToAgentPermissions(
           dataStreamsForPermissions = pkg.data_streams;
           break;
 
+        case 'osquery_manager':
```

Comment:
we could just append this case to L62

Reply:
correct, but as `apm` got a separate case as well, I just tried to follow a pattern :) 
do you want me to squeeze all 3 into a single case?